Bottom: 018959b0417977cc7bf55f309331d69539ab37b3
Top:    147d0b471d9888476ebd3b7fe2ec0a0ee492c8d8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-10 17:44:10 +0100

Refresh of off-to-find-burried-treasure-o

---

diff --git a/src/find.c b/src/find.c
index 9682266..272c205 100644
--- a/src/find.c
+++ b/src/find.c
@@ -4042,7 +4042,7 @@ IsPolygonAnIsland (LayerType *layer, PolygonType *polygon)
 
       /* Check if we got any useful hits */
       connected_count = 0;
-      for (i = 0; i < max_layer; i++)
+      for (i = 0; i < max_copper_layer; i++)
         {
           connected_count += LineList[ i ].Number;
           /* No need to search all layers when one will do */
